### PR TITLE
Fix/notifications focus styles

### PR DESCRIPTION
**Pull request by davemart-in**
## Description

This PR addresses two specific issues:

1) Fix title/cog so that cog cannot be triggered accidentally

Prior to this PR if you clicked anywhere between the title and the cog icon, it triggered the cog link. This should not happen, so I fixed it.

2) Added tab styles to links in popout menu

None of the links in the popout menu had focus styles. I fixed that

**Before**

tab-before

**After**

tab-after

### CODE REVIEW

**Comment by Addison-Stavlo** [comment]
The way this interacts with the current state of notifications shortkeys feels a bit odd.

At the top level, opening with 'n' and using left/right arrows to select a different group we see the shortkey navigation and a11y focus get out of sync.  Similarly here, I don't think there is a way for the tab navigation to select other groups (maybe there doesnt need to be since left/right does, but the focus might need to stay with it).

groupings


In the second level flyout panel, the new a11y focus allows us to highlight items. However, pressing "Enter" still closes the flyout and doesn't trigger the items (which are clickable to lead elsewhere). Currently enter seems toggle the selected item open/closed at this level the same way left/right does and bypasses the interaction on the focused item:

items-in-flyout

Also noticing I cannot reach the comment moderation actions (approve, spam, etc.) with the a11y focus. I think there are shortkeys for every single one of these, but imagine we should be able to get to them in keyboard nav in the end:

Screenshot 2024-10-08 at 9 39 54 AM

**Review by Addison-Stavlo — COMMENTED** [review]
This looks and tests well to me, definitely a step forward!

Minor Q. (also i will smoke test this build on p2 and user sites next)

File: apps/notifications/src/panel/templates/index.jsx [hunk 1]
```diff
@@ -375,8 +375,6 @@ class Layout extends Component {
 					 */
 					activateKeyboard();
 					this.props.selectNote( this.state.selectedNote );
-				} else if ( this.props.selectedNoteId ) {
-					this.props.unselectNote();
```
Comment (Addison-Stavlo) [inline apps/notifications/src/panel/templates/index.jsx:379]: This makes it so the left button no longer closes the panel. Was there a conflict we needed to remove this for?

In general, the change should be fine. But note that /read/notifications the panel opens in the opposite direction and there left previously made sense as the closing direction. We will probably want to create a follow up issue to reverse the left/right shortkeys on /read/notifications at some point in the future?

**Review by Addison-Stavlo — COMMENTED** [review]
This works well on the standalone build as well.

Aside from this PR, I am noticing a small issue on the standalone build (currently on production).

The "close notifications" item in tab flow is slightly cutoff on standalone (testing on p2s or sites directly), but looks good in calypso:

Screenshot 2024-10-08 at 11 16 02 AM
Screenshot 2024-10-08 at 11 16 19 AM

**Review by Addison-Stavlo — APPROVED** [review]
Tests well from my end. My main notes above are for potential other issues to track and resolve separately:

1. for left/right shortkey directions in /read/notifications
2. "close notifications" being partially cutoff in tab flow on the standalone build (p2s / sites)

File: apps/notifications/src/panel/templates/index.jsx (hunk 1 above)
Comment (davemart-in) [reply apps/notifications/src/panel/templates/index.jsx:379]: > In general, the change should be fine. But note that /read/notifications the panel opens in the opposite direction and there left previously made sense as the closing direction. We will probably want to create a follow up issue to reverse the left/right shortkeys on /read/notifications at some point in the future?

Yes, I agree. Let's do this in a follow up issue.

**Pull request merged**